csun-metalab/nomi
# HOTFIX-11-28-2018--clickEventUploadModal

# Description
Added the click event for the upload button and confirm button to be on the whole button instead of just the image.

# Testing
1. Visit any student and click the edit button.
2. Ensure whatever button you press you can now click the edges instead of just the image.
3. Make sure all the functionality isn't broken and is working as intended.

# Installation
You might need to run an npm install to update our vuex and router version
